### PR TITLE
miniamr: add develop version and update release

### PR DESCRIPTION
**Pull request by bhatele**

### CODE REVIEW

File: var/spack/repos/builtin/packages/miniamr/package.py [hunk 1]
```diff
@@ -17,6 +17,9 @@ class Miniamr(MakefilePackage):
 
     tags = ['proxy-app', 'ecp-proxy-app']
 
+    version('develop', branch='master')
```
Comment (junghans) [inline var/spack/repos/builtin/packages/miniamr/package.py]:
```suggestion
    version('develop', branch='master')
    git      = "https://github.com/Mantevo/miniAMR.git"
```

File: var/spack/repos/builtin/packages/miniamr/package.py [hunk 2]
```diff
@@ -13,10 +13,13 @@ class Miniamr(MakefilePackage):
     """
 
     homepage = "https://mantevo.org"
-    url      = "https://github.com/Mantevo/miniAMR/archive/v1.4.0.tar.gz"
+    git      = "https://github.com/Mantevo/miniAMR.git"
```
Comment (junghans) [inline var/spack/repos/builtin/packages/miniamr/package.py]: I think you need both!

Comment (bhatele) [reply var/spack/repos/builtin/packages/miniamr/package.py]: why do we need both?

Comment (junghans) [reply var/spack/repos/builtin/packages/miniamr/package.py]: One for the git checkout e.g. `branch="master"`, one of the tarball download.

Comment (bhatele) [reply var/spack/repos/builtin/packages/miniamr/package.py]: got it, done.

**Pull request merged**